_data/versions.yml

```diff
@@ -1,3 +1,7 @@
+versions:
+  currentReleaseStream: v2.5
+  previousReleaseStream: v2.4
```

Comment:
What is previousReleaseStream used for?

Reply:
Nothing, currently. I've found it useful in some other experiments (swapping new release URLs for htmlproofer), but I'm happy to remove it for now if you prefer.

Reply:
btw, I just https://github.com/projectcalico/calico/pull/932/files - I think this PR removes the need for that one.

Reply:
Yeah, we shouldn't add stuff that isn't used. It's easy to add it in later if we find a real use for it.

I don't understand how this removes the need for the other PR - I think this just changes where the release stream is defined, yeah? There isn't actually any logic that changes here.

Reply:
Removed unused `previousReleaseStream` and moved `currentReleaseStream` to top level in versions.yml.
Re:  https://github.com/projectcalico/calico/pull/932/, you're right, this PR doesn't make a diff to that one.